JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeScreen.kt

```diff
@@ -275,6 +307,7 @@ private fun HomeScreenErrorAndContent(
 @Composable
 private fun PostList(
     posts: List<Post>,
+    state: LazyListState,
```

Comment:
DItto comment about overing a default.

Reply:
Done

Reply:
I don't think this was addressed in PostList

Reply:
You are right, I just fixed it